libs/ledger-live-common/src/families/ethereum/walletApiAdapter.ts

```diff
@@ -0,0 +1,44 @@
+import { EthereumTransaction as WalletAPIEthereumTransaction } from "@ledgerhq/wallet-api-core";
+import {
+  AreFeesProvided,
+  ConvertToLiveTransaction,
+  GetWalletAPITransactionSignFlowInfos,
+} from "../../wallet-api/types";
+import { Transaction } from "./types";
+
+const CAN_EDIT_FEES = true;
+
+const areFeesProvided: AreFeesProvided<WalletAPIEthereumTransaction> = (tx) =>
+  !!(tx.gasLimit || tx.gasPrice);
+
+const convertToLiveTransaction: ConvertToLiveTransaction<
+  WalletAPIEthereumTransaction,
+  Transaction
+> = (tx) => {
+  const hasFeesProvided = areFeesProvided(tx);
+
+  const { gasLimit, ...restTx } = tx;
+
+  const liveTx: Partial<Transaction> = {
+    ...restTx,
+    amount: tx.amount,
+    recipient: tx.recipient,
+    gasPrice: tx.gasPrice,
+    userGasLimit: gasLimit,
+  };
+
+  return hasFeesProvided ? { ...liveTx, feesStrategy: "custom" } : liveTx;
+};
+
+const getWalletAPITransactionSignFlowInfos: GetWalletAPITransactionSignFlowInfos<
```

Comment:
No tests for this one or just an oversight ?

Reply:
Indeed, fixed here -> f8351c25fdf46ee25bdeabac47f9a3db85d46774 ✅